public/app/AppWrapper.tsx

```diff
@@ -107,18 +107,20 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
                 >
                   <ModalsProvider>
                     <GlobalStyles />
-                    {config.featureToggles.commandPalette && <CommandPalette />}
+                    {!config.isPublicDashboardView && config.featureToggles.commandPalette && <CommandPalette />}
                     <div className="grafana-app">
                       <Router history={locationService.getHistory()}>
-                        {ready && <>{newNavigationEnabled ? <NavBarNext /> : <NavBar />}</>}
+                        {!config.isPublicDashboardView && ready && (
+                          <>{newNavigationEnabled ? <NavBarNext /> : <NavBar />}</>
+                        )}
```

Comment:
```suggestion
                        {this.renderNavBar()}
```

Move this to a small render func, should make the logic easier for both if we should render anything and if we should render new or next nav